pkgs/applications/science/misc/openmvs/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchFromGitHub, pkgconfig, cmake
+, eigen, opencv, ceres-solver, cgal, boost, vcg
+, gmp, mpfr, glog, google-gflags, libjpeg_turbo }:
+
+stdenv.mkDerivation rec {
+  version = "a3b3600";
+  name = "openmvs-${version}";
+
+  src = fetchFromGitHub {
+    owner = "cdcseacave";
+    repo = "openmvs";
+
+    rev = "a3b360016660a1397f6eb6c070c2c19bbb4c7590";
```

Comment:
So you did not choose a release because of dependency incompatibilities?
https://github.com/cdcseacave/openMVS/releases